src/components/icons/chromicons.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Link from 'next/link';
 
 export const Chromicons = ({ ...rootProps }) => (
-  <Link href="/" passHref>
+  <Link href={`${process.env.BACKEND_URL}`} passHref>
```

Comment:
I don't believe you need the string and can get away with: 
```suggestion
  <Link href={process.env.BACKEND_URL} passHref>
```

Reply:
Strange, I could've sworn I added the other changes to the commit, guess not! :)